docs/aspenmesh-collector.MD

```diff
@@ -1,50 +1,32 @@
 # Aspen OpenTelemetry Collector
 
-The Aspen OpenTelemetry Collector collects telemetry data from Kubernetes and your Service Mesh and sends it to the [Aspen Mesh Application Intelligence Platform](https://aspenmesh.io/).
+The Aspen OpenTelemetry Collector collects telemetry data from Kubernetes and your Service Mesh and sends it to the [Aspen Mesh Application Intelligence Platform](https://aspenmesh.io/). Our platform is built for modern App and SRE teams to see and understand how their systems are behaving.
 
 ## Installation
 
-### Step 1
-
-Add this repository to your Helm client
+### Add the Aspen Mesh Helm Repository
 
 ```bash
 helm repo add aspenmesh https://aspenmesh.github.io/aspenmesh-charts/
-```
-
-```console
-"aspenmesh" has been added to your repositories
-```
-
-### Step 2
-
-Update your Helm client
-
-```bash
 helm repo update
 ```
 
-```console
-Hang tight while we grab the latest from your chart repositories...
-...Successfully got an update from the "aspenmesh" chart repository
-```
-
-### Step 3
+### Obtain your Aspen Mesh API Key
 
 Note your API key in the [Aspen Mesh dashboard](https://console.cloud.aspenmesh.io/client).
 
 [Register for a new account](https://console.cloud.aspenmesh.io/v1/oauth2/register) if you haven't signed up for Aspen Mesh.
 
-### Step 4
-
-Install the Aspen OpenTelemetry Collector in an existing Kubernetes cluster. You'll need your API key from **Step 3**.
+### Install the Aspen OpenTelemetry Collector
 
 With you API key, execute the following command:
 
 ```bash
 helm upgrade --install aspenmesh-collector aspenmesh/aspenmesh-collector -n aspenmesh --create-namespace --set apiKey=<replace-with-your-api-key>
```

Comment:
Does this need to change?
```suggestion
helm upgrade --install aspen-collector aspenmesh/aspen-collector -n aspenmesh --create-namespace --set apiKey=<replace-with-your-api-key>
```

Reply:
Not yet. There may be end users who have installed the connector. So there need to be some thought behind making that change.

Reply:
This PR renames `aspenmesh/aspenmesh-collector` to `aspenmesh/aspen-collector` right? Won't it break right away?

Reply:
Or does the directory rename not matter?

Reply:
It shouldn't matter as the details are in the chart file, but I will reverse it to be safe.